bridges/snowbridge/primitives/inbound-queue/src/v2/message.rs

```diff
@@ -64,6 +71,23 @@ impl core::fmt::Debug for IGatewayV2::EthereumAsset {
 	}
 }
 
+impl core::fmt::Debug for IGatewayV2::Xcm {
+	fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+		f.debug_struct("Xcm")
+			.field("kind", &self.kind)
+			.field("data", &self.data)
+			.finish()
+	}
+}
+
+#[derive(Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
+pub enum XcmCommand {
```

Comment:
Within this context, I'd prefer `XcmPayload` as a name.

```suggestion
pub enum XcmPayload {
```